torchaudio/datasets/librispeech_biasing.py

```diff
@@ -0,0 +1,134 @@
+import os
+from pathlib import Path
+from typing import Tuple, Union
+
+import torchaudio
+from torch import Tensor
+from torch.hub import download_url_to_file
+from torch.utils.data import Dataset
+from torchaudio.datasets.utils import extract_archive
```

Comment:
`extract_archive` is replaced with `_extract_tar` and `_extract_zip` in #2927. Here we can use `_extract_tar`